src/renderer/components/radio/radio.tsx

```diff
@@ -85,6 +85,7 @@ export function Radio<T>({
         type="radio"
         checked={checked}
         onChange={() => ctx.onSelect(value)}
+        disabled={disabled || ctx.disabled}
```

Comment:
Shouldn't it be like this (since `props.disabled` is optional?)

```suggestion
        disabled={disabled ?? ctx.disabled}
```

Reply:
No because `disabled` is defaulted to `false` above